.travis.yml

```diff
@@ -1,10 +1,18 @@
-language: python
-sudo: false
-python:
-- '2.7'
+#Travis OSX currently doesn't not support docker (via virtualbox).
+# "This computer doesn't have VT-X/AMD-v enabled."
+sudo: required
+dist: trusty
 install:
-  - bin/install.sh --ci
+  - sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+  - sudo sh -c "echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list"
+  - sudo apt-get update -qq
+  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y -q docker-engine
+  - sudo curl -o /usr/local/bin/docker-compose -L https://github.com/docker/compose/releases/download/1.6.2/docker-compose-`uname -s`-`uname -m`
+  - sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
In general I prefer to keep the lines of vendor-specific CI files sparse, and instead throw these lines in a script where it's easier to document, maintain and understand. `bin/ci-install-docker-ecosystem.sh`?

Not a blocker for merge.